content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+{{< text bash >}}
+$ kubectl create secret generic ${CLUSTER_NAME} --from-file ${WORK_DIR}/${CLUSTER_NAME} -n ${NAMESPACE}
+$ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
+{{< /text >}}
+
+## Deploy Bookinfo Example Across Clusters
+
+1.  Install bookinfo on the first cluster.  Remove the `reviews-v3` deployment to deploy on remote.
+
+    {{< text bash >}}
+    $ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+    $ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+    $ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+    $ kubectl delete deployment reviews-v3
+    {{< /text >}}
+
+1.  Install the reviews-v3 deployment on the remote.  _Note:_ The `ratings` service definition is added
+    to the remote cluster because `reviews-v3` is a client of `ratings` and creating the service object
+    creates a DNS entry.  The Istio sidecar in the `reviews-v3` pod will determine the proper `ratings`
+    endpoint after the DNS lookup is resolved to a service address.
+
+    {{< text bash >}}
+    $ cat <<EOF > ~/tmp/reviews-v3.yaml
+    ---
+    ##################################################################################################
+    # Ratings service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: ratings
+      labels:
+        app: ratings
+    spec:
+      ports:
+      - port: 9080
+        name: http
+    ---
+    ##################################################################################################
+    # Reviews service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: reviews
+      labels:
+        app: reviews
+    spec:
+      ports:
+      - port: 9080
+        name: http
+      selector:
+        app: reviews
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: reviews-v3
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: reviews
+            version: v3
+        spec:
+          containers:
+          - name: reviews
+            image: istio/examples-bookinfo-reviews-v3:1.5.0
+            imagePullPolicy: IfNotPresent
+            ports:
+            - containerPort: 9080
+    EOF
+    {{< /text >}}
+
+    {{< text bash >}}
+    $ kubectl config use-context "gke_${proj}_${zone}_cluster-2"
+    $ kube apply -f ~/tmp/reviews-v3.yaml
+    {{< /text >}}
+
+1.  Get the `istio-ingressgateway` service's external IP to access the `bookinfo` page to validate that all
+    reviews stars are seen.
+
+    {{< text bash >}}
+    $ kubectl get svc istio-ingressgateway -n istio-system
+    {{< /text >}}
+
+    Access `http://<GATEWAY_IP>/productpage` repeatedly and each version of reviews should be equally loadbalanced,
```

Comment:
Other portions of the documentation state it can take up to 15 times to obtain a fully load balanced result.  Might just remove equally and offer some information that it may take multiple retries to obtain a fully load balanced operation.

Reply:
ok cool... I'll wordsmith it a bit.